lang/en.json

```diff
@@ -502,7 +502,7 @@
     "forms": {
         "range": "You need to enter numbers only between: {limitTo} and {limitFrom}",
         "contact_us": {
-            "full_name": "Full Name",
+            "full_name": "Nom complet",
```

Comment:
@CyrilAtHome did you mean to modify `fr.json` file ?

Reply:
Hi @junedkazi , till now i just want to test if a modification of en.json has a real impact on my website at bigcommerce.com.
But perhaps i didn't undestand the purpose of github.com :(

Reply:
OK, after reading the Guide, i understood that Github is a platform for learning how to develop correctly.
I have a website on bigcommerce.com, and i have to localize it by modifying fr.json, but i didn't understand how to push it on bigcommerce.com. It seems that i have to install stencil CLI, but the doc on bigcommerce.com was not update with the last version of all modules needed and the install doesn't work.